velox/exec/tests/HashJoinTest.cpp

```diff
@@ -5751,6 +5751,49 @@ DEBUG_ONLY_TEST_F(HashJoinTest, hashBuildAbortDuringInputgProcessing) {
   }
 }
 
+TEST_F(HashJoinTest, malformedDictEvalFilter) {
+  /*
+   * The last row of previous batch did pass the filter.
```

Comment:
The last row of the previous batch did `not` pass the filter (false).

This should be the case if it needs to be the first row of the next batch.